charts/pip-frontend/values.yaml

```diff
@@ -7,6 +7,7 @@ nodejs:
   environment:
     OIDC: true
     AUTH_RETURN_URL: https://127.0.0.1/login/return
+    MEDIA_VERIFICATION_RETURN_URL: https://127.0.0.1/media-verification/return
```

Comment:
This will fail in prod non dev or staging environments, as it is not overridden. Needs putting in sds-flux-config :)

Reply:
Good point. Have added them, see: https://github.com/hmcts/sds-flux-config/pull/2173